charts/payment-integration/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "payment.fullname" . }}
+  labels:
+    app: {{ template "payment.name" . }}
+    chart: {{ template "payment.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ default 2 .Values.replicaCount }}
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: {{ template "payment.name" . }}
```

Comment:
label `release` should be added here too
```
selector:
  matchLabels:
    app: {{ template "payment.name" . }}
    release: {{ .Release.Name }}
```

Reply:
eec622b